lib/pipeline/addon/pipelines/project/pipeline/route.js

```diff
@@ -0,0 +1,76 @@
+import Ember from 'ember';
+import { inject as service } from '@ember/service';
+
+export default Ember.Route.extend({
+  globalStore: service(),
+  projectStore: service('store'),
+  scope: service(),
+  queryParams: {
+    mode: {
+      refreshModel: true
+    },
+  },
+  model: function(params) {
+    let globalStore = this.get('globalStore');
+    let projectStore = this.get('projectStore');
+    let scope = this.get('scope');
+    let projectId = scope.currentProject&&scope.currentProject.id;
```

Comment:
Use modelFor() to get one you want from the project/cluster route.  Avoid touching `scope`.